KalturaWowzaServer/src/main/java/com/kaltura/media_server/modules/AuthenticationModule.java

```diff
@@ -87,7 +91,12 @@ private KalturaLiveEntry onClientConnect(WMSProperties properties, HashMap<Strin
 
     public void onDisconnect(IClient client) {
         try{
-            KalturaLiveEntry liveEntry = Utils.getKalturaLiveEntry(client);
+            KalturaEntryIdKey entryIdKey = null;
```

Comment:
you are suppose to remove the KALTURA_ENTRY_DATA_PERSISTENCY_KEY 
not sure why this code